pkg/btf/btf_test.go

```diff
@@ -178,7 +178,16 @@ func genericTestFindBtfFuncParamFromHook(t *testing.T, hook string, argIndex int
 	return nil
 }
 
-func TestFindBtfFuncParamFromHook(t *testing.T) {
+func RunTestOnSingleBtf(t *testing.T, btfFname string, test func(t *testing.T)) {
+	if err := btfFileExists(btfFname); err != nil {
+		t.Skip(err)
```

Comment:
If the BTF file does not exist at the specified path, the test is skipped. This means that those tests will be ignored for kernel 4.9 without complexity. But the downside of this, is if the path is wrong, the test is ignored, and we will not be noticed